docs/collections/view-collection-settings.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebar_position: 20
+title: View Collection Settings
+---
+
+import ViewCollectionSettings from './_partials/_view-collection-settings.md';
+import CollectionSettingsApi from './_partials/_collection-settings-api.md';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page describes how to view and update collection settings.
+
+<Tabs groupId="operating-systems">
+<TabItem value="ui" label="UI">
+
+In the Macrometa web console, you can view and update collection settings in the Settings tab. You can also [Delete](delete-collection.md) or [Truncate](truncate-collection.md) the collection.
+
+<ViewCollectionSettings />
+
+The Settings tab contains the following fields:
+
+**Collection Details**
+
+- **Collection Name** - The name set at collection creation. Cannot be changed.
+- **Collection ID** - System-generated ID of the selected collection.
+- **Distribution** - Global or local, defined at collection creation. For more information, refer to [Global or Local Collections](./index.md#global-or-local-collections).
+- **Data Model** - Type of data that the collection accepts, either **document** or **edge**.
+- **Synchronous Writes** - Click to turn synchronous writes on or off for the collection.
+- **Resource URL** - API URL and endpoint to access the collection. You might use this when creating a cURL command or HTTP request to interact with the collection.
+
+![Collection Details Section](/img/collections/collection-details.png)
+
+**Collection Stream**
+
+This section is not in [Redis Mode collection](./redis-mode/) settings.
+
+- **Stream Enabled** - Click to turn streams on or off for this collection. When streams are enabled, you can access the Stream tab.
+- **Stream Name** - The name of the collection stream.
+- **Replication** - Whether the stream is replicated globally or locally.
+- **WebSocket URL** - The WebSocket connection to the collection stream allows you to interact with the stream in real-time. This connection allows you to receive live updates as documents are added, updated, or removed from the collection.
+
+![Collection Stream Section](/img/collections/collection-stream.png)
+
+**Connections and Reload Collections**
+
+The Connections and Reload Collections sections only apply to [Document Store collections](./documents/). For more information about integrations and connections, refer to [Integrations](../integrations/).
```

Comment:
@diana-macrometa - Shall we mention somewhere about the behavior of the Reload Collections? It initially truncates the current collection and then only re-starts the Source connection.

Reply:
I did not know that. I was planning to put that sort of information in Integrations, but maybe it makes more sense here.